spec-tests/src/test/java/org/candlepin/spec/hypervisors/HypervisorCheckInSpecTest.java

```diff
@@ -1706,7 +1707,7 @@ private PoolDTO findDerivedPoolForHost(String ownerKey, String type, String prod
         return null;
     }
 
-    private class CommonTestData {
+    public static class CommonTestData {
```

Comment:
I'm not sure that making this public and references in other test suites is a good idea. Someone working in HypervisorCheckInSpecTest could change this class and break other tests suites without realizing it. If this class should be used by multiple test suites it should probably live in the org.candlepin.spec.bootstrap.data.builder package, but it would probably need a different name since the data isn't necessarily common for all test suites. Maybe something like 'HypervisorData'?

Reply:
I'd like to get some feedback from others on this as well. @nikosmoum @Ceiu @Januson

Reply:
Being in its own file and named HypervisorTestData as @wottop has currently done looks good to me